cluster/src/multi-jvm/resources/logback-test.xml

```diff
@@ -0,0 +1,13 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="info">
+        <appender-ref ref="STDOUT" />
+    </root>
+    <logger name="io.netty.util.Recycler" level="ERROR" />
```

Comment:
Is this something that was needed just because of netty4 or is it generally helpful? If it is generally helpful I would do this change in a separate PR so that we can backport it to Pekko 1.0.x

Reply:
It because there are so much netty log when you run testing.

Reply:
I thought so, do you mind making this change in a separate PR so it can be back-ported separately to Pekko 1.0.x?

I can also do this, and technically speaking you shouldn't even need to update the PR since it would get resolve via a merge.

Reply:
There was a miscommunication, I realize that `PoolThreadCache` and `Recycler` classes only exist in Netty 4 and not Netty 3 (just checked now), resolving conversation.